gsoc/2021/mentors.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Summary of Mentors GSoC 2021
+layout: plain
+---
+
+## Full Mentor List (Name, Email, Org)
+
+* John Doe [jdoe@domain](mailto:jdoe@domain) Institution Name (this is an example, entries to be sorted in last name alphabetic order)
```

Comment:
I would enforce the alphabetic order constraint: 
  "entries must be sorted in last name alphabetic order"